modules/introduction/partials/new_features-76_4.adoc

```diff
@@ -30,6 +30,12 @@ curl --get -u <username:password> \
 [#new-features-764-search-service]
 === Search Service
 
+* The Search Service now supports pre-filtering on vector search queries. 
+Pre-filtering allows you to execute vector searches over a subset of the vector index, 
+which can reduce the computational resource required to complete the query.
```

Comment:
It does not reduce the compute required for the query. Let's simply replace this with ..

```
* The Search Service now supports pre-filtering on vector search queries. 
Pre-filtering allows you to execute vector searches over a subset of the vector index, 
via the means of a filter request that qualifies the subset.
```